README.md

```diff
@@ -26,3 +28,10 @@
 
 ---
 ## Steps to migrate provider
+- Clone the github repository
+- Give permissions to the script using chmod +x migrate.sh
```

Comment:
Are you sure giving permission to only one script will work?

Reply:
Yes that is good enough